docs/blog/write-the-docs-2022-stats.rst

```diff
@@ -0,0 +1,87 @@
+.. post:: Jan 13, 2023
+   :tags: stats, year in review
+   :author: Eric Holscher
+
+Write the Docs 2022 Stats
+=========================
+
+This year we have done a :doc:`/blog/2023-january-update` post along with our stats, which has more context and information about where we're headed in 2023.
+
+Unfortunately our Podcast has been in hiatus for a while with no new episodes in 2022,
+so we have removed it from the stats post.
+
+2022 Stats
+----------
+
+We have shared our high-level community stats for the past 5 years,
+and will continue to do it each year going forward.
+
+You can read our previous posts from 2016_, 2017_, 2018_, 2019_, 2020_, 2022_.
+
+.. _2022: https://www.writethedocs.org/blog/write-the-docs-2022-stats/
```

Comment:
```suggestion
We have shared our high-level community stats since 2016,
and will continue to do it each year going forward.

You can read our previous posts from 2016_, 2017_, 2018_, 2019_, 2020_, _2021_, 2022_.

.. _2022: https://www.writethedocs.org/blog/write-the-docs-2022-stats/
.. _2022: https://www.writethedocs.org/blog/write-the-docs-2021-stats/
```

Reply:
Amazing how we kinda just kept doing stuff in the last couple of years. Go past us!